### ISSUE TITLE
edge/Win10 after d&d mouse click does not responding

### ISSUE THREAD

**Issue by vavdav** [gaearon/react-dnd#390]
I have problem with win10 and edge any another browser works correctly. After drag & drop clicking on everything does not working. Does anybody know how to solve it?

**Comment by BugsBunny338** [comment]
Ok, we finally figured out what was the problem. For anyone experiencing the same issue:

In MSEdge (and it seems ONLY in MSEdge; other IE versions, Firefox, Chrome, Safari, ... worked fine) the whole UI of our React application stopped responding. No error message, nothing, the page just went dead.

We implemented our `DropHere` component as such:
```
import { DropTarget } from 'react-dnd';

class DropHere {
   /* ... code shortened for brevity ... */
  render() {
    const { canDrop } = this.props;
    return canDrop ? <div>drop here</div> : false;
  }
}

export default DropTarget(types, spec, collect)(DropHere);
```

So that dropzone only appears while dragging. What fixed the problem with frozen UI was to **always render the dropzone `div` and hide it with CSS** when nothing is being dragged.
